integration/bin/alluxio-yarn.sh

```diff
@@ -41,6 +42,8 @@ source "${SCRIPT_DIR}/common.sh"
 
 NUM_WORKERS=$1
 HDFS_PATH=$2
+MASTER_ADDRESS=$3
```

Comment:
how about having the condition here 
`MASTER_ADDRESS=${3:-ALLUXIO_MASTER_HOSTNAME}`